cockatrice/src/dlg_update.cpp

```diff
@@ -22,8 +22,8 @@ DlgUpdate::DlgUpdate(QWidget *parent) : QDialog(parent) {
     progress = new QProgressBar(this);
 
     QDialogButtonBox *buttonBox = new QDialogButtonBox(this);
-    ok = new QPushButton("Ok", this);
-    manualDownload = new QPushButton(tr("Update Anyway"), this);
+    ok = new QPushButton("Close", this);
+    manualDownload = new QPushButton(tr("Download Anyway"), this);
```

Comment:
I liked the original `Update` wording, because it triggers a download+install = update.
It's not just a simple download.